### PR TITLE
Update FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql

### PR DESCRIPTION
**Pull request by sailendrasingh1**
Detail: Code Update for September Release for Scrambled Data 
Date:10-06-2023

### CODE REVIEW

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql [hunk 1]
```diff
@@ -212,71 +401,124 @@ END
...
++ '|' + ISNULL(FNRL_CITY,'')			--249
++ '|' + ISNULL(FNRL_STATE,'')			--250
++ '|' + ISNULL(FNRL_ZIPCD,'')			--251 
++ '|' + ISNULL(FNRL_CNTRY,'')			--252 
++ '|' + ISNULL(DISP_NME,'')				--253
++ '|' 									--254
++ '|' + ISNULL(DISP_CTY,'')				--255
++ '|' + ISNULL(DISP_ST,'')				--256
++ '|' + ISNULL(DISP_CNTRY,'')			--267
++ '|' + ISNULL(SFN_NUM_OOS,'')			--258
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:520]: should be blank

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql [hunk 2]
```diff
@@ -300,28 +542,90 @@ END
 					THEN ISNULL((CONVERT(VARCHAR(30),DATEADD(DAY,+2,CONVERT (DATETIME , BURIAL_PERMIT_ISSUE_DATE )),112 ) + '6666'),'')
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:542]: CASE WHEN C.CURRENT_CITY_OPT_IN = 'Y' AND BURIAL_PERMIT_ISSUE_DATE IS NULL AND **ISDATE(BURIAL_PERMIT_ISSUE_DATE_RV) = 0** (Why)?
					THEN REPLACE ((RIGHT(BURIAL_PERMIT_ISSUE_DATE,4) + LEFT(BURIAL_PERMIT_ISSUE_DATE,6)),'/','') + '6666' - Is this correct?

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql [hunk 3]
```diff
@@ -300,28 +542,90 @@ END
...
+			 WHEN BURIAL_PERMIT_NONE = 'N' THEN
+				(CASE WHEN C.CURRENT_CITY_OPT_IN = 'Y' THEN ISNULL(BURIAL_PERMIT_NUMBER,BURIAL_PERMIT_NUMBER_RV)
+				 ELSE ISNULL(BURIAL_PERMIT_NUMBER,'') END)
 		
-		END
-+ '|' + OCCUR_REGIS_NUM + '||||' 
-+ CASE WHEN EXP_SSA_DATE_CREATED IS NULL THEN '' 
-		ELSE CONVERT(VARCHAR(30),DATEADD(DAY,+2,CONVERT (DATETIME , EXP_SSA_DATE_CREATED)),112 ) + '' +
+		END					--260
++ '|' + OCCUR_REGIS_NUM		--261				--REVISITED IN SEPT RELEASE
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:553]: ISNULL is missing for OCCUR_REGIS_NUM

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql [hunk 4]
```diff
@@ -1216,25 +1521,53 @@ END
...
+		END		--394
 + '|' 
 --+ CASE WHEN DOI = '99/99/9999' THEN '' 
 --  ELSE ISNULL(CONVERT(VARCHAR(30),DATEADD(DAY,+2,CONVERT (DATETIME, DOI)),112),'') END
 --+ REPLACE(ISNULL(TOI,''),':','') 
-+ CASE WHEN DOI = '99/99/9999' THEN ''
++ CASE --WHEN DOI = '99/99/9999' THEN '999999996666'				--commented out according to code review 07/07/2023
 	   WHEN DOI IS NULL THEN ''
+	   WHEN ISDATE(DOI)= 0 AND REPLACE(DOI,'9','') IN ('/', '') THEN '99999999' + CONVERT( VARCHAR,REPLACE(ISNULL(TOI,'6666'),':',''))    -- Check MAc incorrect value for DOI it db
 	   WHEN ISDATE(DOI)= 0 THEN  REPLACE ((RIGHT(DOI,4) + LEFT(DOI,6)),'/','') + REPLACE(ISNULL(TOI,'6666'),':','')
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:1562]: WHEN ISDATE(DOI)= 0 THEN  REPLACE ((RIGHT(DOI,4) + LEFT(DOI,6)),'/','') + REPLACE(ISNULL(TOI,'6666'),':','') 
The same conversion should be applied for the section TOI

which is similar to 
  WHEN ISDATE(DOI) = 1 THEN ISNULL(CONVERT(VARCHAR(30),DATEADD(DAY,+2,CONVERT (DATETIME, DOI)),112),'') 
							+ (CASE WHEN TOI_IND = 'D' AND TOI = '12:00' THEN '0000'
								ELSE CAST(
											(CASE WHEN TOI_IND = 'P'  AND TOI NOT LIKE '12:%' THEN CAST(CAST(LEFT(REPLACE(TOI,':',''),2) AS INT)+12 AS VARCHAR(100)) +CAST(RIGHT(REPLACE(TOI,':',''),2) AS VARCHAR(100))
											   ELSE REPLACE(ISNULL(TOI,'6666'),':','') 
											 END) AS VARCHAR(10))
								END)

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql [hunk 5]
```diff
@@ -1245,22 +1578,66 @@ END
...
++ '|' + CONVERT (VARCHAR(8), DATEADD(DAY,+2,CONVERT(DATETIME, VRV_DATE_CHANGED)),112) + '' + REPLACE(LEFT(CAST(CONVERT (TIME , VRV_DATE_CHANGED) AS VARCHAR (20)),6),':','')		--410
++ '|' + 'Registered'					--411
++ '|'									--412			
++ '|'									--413
++ '|'									--414
++ '|'									--415
++ '|'									--416
++ '|'									--417
++ '|'									--418
++ '|' + 'REGISTERED'					--419				--bezawit will check
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:1623]: Did you remember did i checked this as per your comment **"Bezawit will check this"**. I don't see this the mapping

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql [hunk 6]
```diff
@@ -1304,56 +1681,863 @@ END
...
++ '|'										--500
++ '|' + CASE WHEN TOI_IND = 'A' THEN 'AM'
+			 WHEN TOI_IND = 'P' THEN 'PM'
+			 WHEN TOI_IND = 'M' THEN 'Military'
+			 WHEN TOI_IND = 'U' THEN 'Unknown'
+			 WHEN TOI_IND IS NOT NULL AND TOI = '99:99' THEN ''		
+			 WHEN TOI_IND IS NULL THEN ''		
+			 WHEN TOI = '12:00' AND TOI_IND = 'D' THEN 'AM'
+			 WHEN TOI = '12:00' AND TOI_IND = 'N' THEN 'PM'
+			 WHEN TOI = '99:99' THEN ''
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:1751]: Is TOI converted to Military? Should not this be converted to Military? We need to check this?

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql [hunk 7]
```diff
@@ -1,93 +1,170 @@
...
+/*************************************************THIS SECTION IS TO GET THE OUTPUT FILE*************************************************************************/
+
+SELECT DEATH_REC_ID,D.INTERNAL_CASE_NUMBER,SFN_NUM
+,(CONVERT(VARCHAR(30),DATEADD(DAY,+2,CONVERT(DATETIME , DOD_4_FD)),112 )
 + (CASE WHEN CERT_DESIG = 'MEDICAL EXAMINER' THEN
 			CAST((CASE WHEN TOD_IN_ME = 'P' AND TOD_ME NOT LIKE '12:%' THEN CAST(CAST(LEFT(REPLACE(TOD_ME,':',''),2) AS INT)+12 AS VARCHAR(100)) +CAST(RIGHT(REPLACE(TOD_ME,':',''),2) AS VARCHAR(100))
-						WHEN TOD_IN_ME = 'D' AND TOD_ME = '12:00' THEN '0000'
-						ELSE REPLACE(ISNULL(TOD_ME,'6666'),':','') 
-						END) AS VARCHAR(10))
+					   WHEN TOD_IN_ME = 'D' AND TOD_ME = '12:00' THEN '0000'
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:56]: I think this is wrong. 1200 is noon, 0000 is midnight
  WHEN TOD_IN_ME = 'D' AND TOD_ME = '12:00' THEN '0000' = 1200 is noon
there should be another one 
TOD_IN_ME = 'M' AND TOD_ME = '12:00'   = 0000 is midnight

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql [hunk 8]
```diff
@@ -1,93 +1,170 @@
...
+					   ELSE REPLACE(ISNULL(TOD_ME,'6666'),':','') 
+				  END) AS VARCHAR(10))
 			WHEN TOD_IN = 'D' AND TOD = '12:00' THEN '0000'
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:59]: Same as above

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql [hunk 9]
```diff
@@ -1216,25 +1521,53 @@ END
...
-+ CASE WHEN DOI = '99/99/9999' THEN ''
++ CASE --WHEN DOI = '99/99/9999' THEN '999999996666'				--commented out according to code review 07/07/2023
 	   WHEN DOI IS NULL THEN ''
+	   WHEN ISDATE(DOI)= 0 AND REPLACE(DOI,'9','') IN ('/', '') THEN '99999999' + CONVERT( VARCHAR,REPLACE(ISNULL(TOI,'6666'),':',''))    -- Check MAc incorrect value for DOI it db
 	   WHEN ISDATE(DOI)= 0 THEN  REPLACE ((RIGHT(DOI,4) + LEFT(DOI,6)),'/','') + REPLACE(ISNULL(TOI,'6666'),':','') 
-	   WHEN ISDATE(DOI) = 1 THEN ISNULL(CONVERT(VARCHAR(30),DATEADD(DAY,+2,CONVERT (DATETIME, DOI)),112),'')  + REPLACE(ISNULL(TOI,'6666'),':','')
-  END
+	   WHEN ISDATE(DOI) = 1 THEN ISNULL(CONVERT(VARCHAR(30),DATEADD(DAY,+2,CONVERT (DATETIME, DOI)),112),'') 
+							+ (CASE WHEN TOI_IND = 'D' AND TOI = '12:00' THEN '0000'
+								ELSE CAST(
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:1565]: Same as above

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql [hunk 10]
```diff
@@ -1304,56 +1681,863 @@ END
...
++ '|'									--733
++ '|'									--734
++ '|'									--735
++ '|'									--736
++ '|'									--737
++ '|'									--738
++ '|'									--739
++ '|'									--740
++ '|'									--741
++ '|' +  ISNULL(CONVERT(VARCHAR(30),DATEADD(DAY,+2,CONVERT (DATETIME, OCCUR_REGIS_DATE)),112),'') + '6666' --742
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:1996]: Are all the dates date?

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql [hunk 11]
```diff
@@ -1304,56 +1681,863 @@ END
...
++ '|'														--773
++ '|' + CASE WHEN ATTEND_PHYSICIAN_NONE = 'Y' THEN '1'
+			 WHEN ATTEND_PHYSICIAN_NONE = 'N' THEN '0'
+		END													--774  
++ '|' + ISNULL(NP_PHYSICIAN_ASSOCIATION,'')					--775
++ '|' + ISNULL(PH.Mapping_Current_PhysicanFirstName,'')		--776
++ '|' + ISNULL(PH.Mapping_Current_PhysicanLastName,'')		--777
++ '|' + ISNULL(PH.Mapping_Current_PhysicanMiddleName,'')	--778
++ '|' + ISNULL(PH.Mapping_Current_PhysicanSuffix,'')		--779
++ '|' + ISNULL(PH.Mapping_Current_PhysicanTitle,'')			--780
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:2042]: is the  title from att-c?

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql [hunk 12]
```diff
@@ -1304,56 +1681,863 @@ END
...
++ '|' + CASE WHEN VET1_DATE_ENTERED = '99/99/9999' THEN '999999996666' --SHOULD THIS BE 99999999 OR LEFT BLANK IF YES THEN RECHECK FOR DOI
+			 WHEN VET1_DATE_ENTERED IS NULL THEN ''
+			 --WHEN VET1_DATE_ENTERED LIKE 'N%A' THEN '666666666666'
+			 WHEN ISDATE(VET1_DATE_ENTERED)= 0 AND REPLACE(VET1_DATE_ENTERED,'9','') IN ('/', '') THEN '99999999' + '6666'   -- dont need toi part only keep 6666
+			 WHEN ISDATE(VET1_DATE_ENTERED)= 0 THEN  REPLACE ((RIGHT(VET1_DATE_ENTERED,4) + LEFT(VET1_DATE_ENTERED,6)),'/','') + '6666'
+			 WHEN ISDATE(VET1_DATE_ENTERED) = 1 THEN ISNULL(CONVERT(VARCHAR(30),DATEADD(DAY,+2,CONVERT (DATETIME, VET1_DATE_ENTERED)),112),'')  + '6666'
+		END													--792
++ '|' + CASE WHEN VET1_DATE_DISCHARGED = '99/99/9999' THEN '999999999999' --SHOULD THIS BE 99999999 OR LEFT BLANK IF YES THEN RECHECK FOR DOI
+			 WHEN VET1_DATE_DISCHARGED IS NULL THEN ''
+			 WHEN VET1_DATE_DISCHARGED LIKE 'N%A' THEN '666666666666'
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:2090]: Why '666666666666' it should be 999999996666

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql [hunk 13]
```diff
@@ -1304,56 +1681,863 @@ END
...
++ '|' + CASE WHEN VET2_DATE_ENTERED = '99/99/9999' THEN '999999996666'
+			 WHEN VET2_DATE_ENTERED IS NULL THEN ''
+			-- WHEN VET2_DATE_ENTERED LIKE 'N%A' THEN '666666666666'
+			 WHEN ISDATE(VET2_DATE_ENTERED)= 0 AND REPLACE(VET2_DATE_ENTERED,'9','') IN ('/', '') THEN '99999999' + '6666'   -- Check MAc incorrect value for DOI it db
+			 WHEN ISDATE(VET2_DATE_ENTERED)= 0 THEN  REPLACE ((RIGHT(VET2_DATE_ENTERED,4) + LEFT(VET2_DATE_ENTERED,6)),'/','') + '6666'
+			 WHEN ISDATE(VET2_DATE_ENTERED) = 1 THEN ISNULL(CONVERT(VARCHAR(30),DATEADD(DAY,+2,CONVERT (DATETIME, VET2_DATE_ENTERED)),112),'')  + '6666'
+	    END													--799
++ '|' + CASE WHEN VET2_DATE_DISCHARGED = '99/99/9999' THEN '999999996666'
+			 WHEN VET2_DATE_DISCHARGED IS NULL THEN ''
+			 WHEN VET2_DATE_DISCHARGED LIKE 'N%A' THEN '666666666666'
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:2136]: same as above

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql [hunk 14]
```diff
@@ -1304,56 +1681,863 @@ END
...
++ '|' + CASE WHEN VET3_DATE_ENTERED= '99/99/9999' THEN '999999996666'
+			 WHEN VET3_DATE_ENTERED IS NULL THEN ''
+			-- WHEN VET3_DATE_ENTERED LIKE 'N%A' THEN '666666666666'
+			 WHEN ISDATE(VET3_DATE_ENTERED)= 0 AND REPLACE(VET3_DATE_ENTERED,'9','') IN ('/', '') THEN '99999999' + '6666'   -- Check MAc incorrect value for DOI it db
+			 WHEN ISDATE(VET3_DATE_ENTERED)= 0 THEN  REPLACE ((RIGHT(VET3_DATE_ENTERED,4) + LEFT(VET3_DATE_ENTERED,6)),'/','') + '6666'
+			 WHEN ISDATE(VET3_DATE_ENTERED) = 1 THEN ISNULL(CONVERT(VARCHAR(30),DATEADD(DAY,+2,CONVERT (DATETIME, VET3_DATE_ENTERED)),112),'')  + '6666'
+	    END													--806
++ '|' + CASE WHEN VET3_DATE_DISCHARGED = '99/99/9999' THEN '999999996666'
+			 WHEN VET3_DATE_DISCHARGED IS NULL THEN ''
+			 WHEN VET3_DATE_DISCHARGED LIKE 'N%A' THEN '666666666666'
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:2182]: Same as above

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql [hunk 15]
```diff
@@ -1304,56 +1681,863 @@ END
...
++ '|'					--850
++ '|'					--851
++ '|'					--852
++ '|'					--853
++ '|'					--854
++ '|'					--855
++ '|'					--856
++ '|' + CASE WHEN FL_COD_PENDING = 'Y' THEN '1'
+			 WHEN FL_COD_PENDING = 'N' THEN '0'
+		END				--857
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:2238]: FL_COD_PENDING  can be NULL

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql [hunk 16]
```diff
@@ -1304,56 +1681,863 @@ END
...
++ '|' + D.INTERNAL_CASE_NUMBER						--979	PART OF SEPT RELEASE
++ '|' + ISNULL(ST_PREVIOUS_VOL_PAGE,'')				--980	PART OF SEPT RELEASE
++ '|' + CAST(DEATH_REC_ID AS VARCHAR)								--981	PART OF SEPT RELEASE
++ '|' + CASE WHEN D.RESIDE_REGIS_NAMEL IS NOT NULL 
+				THEN + ISNULL(ORN2.Mapping_Current_RegistrarFirstName,'') + ';' 
+					 + ISNULL(ORN2.Mapping_Current_RegistrarMiddleName,'') + ';' 
+					 + ISNULL(ORN2.Mapping_Current_RegistrarLastName,'') + ';' 
+					 + ISNULL(ORN2.Mapping_Current_RegistrarSuffix,'')	
+			 WHEN D.RESIDE_REGIS_NAMEL IS NULL 
+				THEN ISNULL(CTC2.CLERK_FIRST_NAME,'')+ ';' + ISNULL(CTC2.CLERK_MIDDLE_NAME,'') +';' +ISNULL(CTC2.CLERK_LAST_NAME,'')+ ';' +ISNULL(CTC2.CLERK_SUFFIX,'')
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:2502]: D.RESIDE_REGIS_NAMEL IS NULL AND D.RESIDE_REGIS_NAME IS NOT NULL

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql [hunk 17]
```diff
@@ -1304,56 +1681,863 @@ END
...
+LEFT JOIN [DAVE_MIGRATION].[VIP_VT_City_Town_Clerk_CD] CTC1 ON CTC1.VT_CITY_TOWN_CLERK_ID=D.OCCUR_REGIS_NAME
+LEFT JOIN [DAVE_MIGRATION].[VIP_VT_City_Town_Clerk_CD] CTC2 ON CTC2.VT_CITY_TOWN_CLERK_ID=D.RESIDE_REGIS_NAME
 WHERE VRV_REGISTERED_FLAG =1
 AND VRV_RECORD_TYPE_ID = '040'
-AND FL_CURRENT =1
+AND D.FL_CURRENT =1
 AND FL_VOIDED=0
-AND ISDATE(DOD)=1
-AND (DOD LIKE '%2018') 
+AND DEATH_REC_ID IN (227382,502407)
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:2533]: Remove this

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql [hunk 18]
```diff
@@ -1304,56 +1681,863 @@ END
...
++ '|'			--978
++ '|' + D.INTERNAL_CASE_NUMBER						--979	PART OF SEPT RELEASE
++ '|' + ISNULL(ST_PREVIOUS_VOL_PAGE,'')				--980	PART OF SEPT RELEASE
++ '|' + CAST(DEATH_REC_ID AS VARCHAR)								--981	PART OF SEPT RELEASE
++ '|' + CASE WHEN D.RESIDE_REGIS_NAMEL IS NOT NULL 
+				THEN + ISNULL(ORN2.Mapping_Current_RegistrarFirstName,'') + ';' 
+					 + ISNULL(ORN2.Mapping_Current_RegistrarMiddleName,'') + ';' 
+					 + ISNULL(ORN2.Mapping_Current_RegistrarLastName,'') + ';' 
+					 + ISNULL(ORN2.Mapping_Current_RegistrarSuffix,'')	
+			 WHEN D.RESIDE_REGIS_NAMEL IS NULL
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:2501]: this should be  WHEN D.RESIDE_REGIS_NAMEL IS NOT NULL AND  D.RESIDE_REGIS_NAMEL IS NULL

**Review by almabekwoldesenbet — CHANGES_REQUESTED** [review]
I checked almost all. The most important one is TOD and TOI that must be checked

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql (hunk 4 above)
Comment (sailendrasingh1) [reply RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:1562]: D is midnight and N is noon according to the code table. So no changes in code

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql (hunk 5 above)
Comment (sailendrasingh1) [reply RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:1623]: Discussed and Fixed

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql (hunk 7 above)
Comment (sailendrasingh1) [reply RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:56]: Discussed and change is not required

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql (hunk 9 above)
Comment (sailendrasingh1) [reply RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:1565]: Discussed and change is not required

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql (hunk 11 above)
Comment (sailendrasingh1) [reply RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:2042]: Created a separated conversion table for this logic

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql (hunk 12 above)
Comment (sailendrasingh1) [reply RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:2090]: Verified using UI

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql (hunk 17 above)
Comment (sailendrasingh1) [reply RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:2533]: will be removed in Final Version

File: RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql (hunk 18 above)
Comment (sailendrasingh1) [reply RVRS_Staging/Database/Procedure/FOR_MASTER_GENERIC_DATA_SCRAMBLING.sql:2501]: FIXED

**Review by almabekwoldesenbet — APPROVED** [review]
looks good

**Pull request merged**